app_create.py

```diff
@@ -82,8 +82,10 @@ def create_app():
     app = cors(
         app, allow_origin=app.config["MEMBRANE_ALLOWED_ORIGINS"], allow_credentials=True
     )
-
-    logging.basicConfig(level=getattr(logging, app.config["MEMBRANE_LOGGING_LEVEL"]))
+    logging.basicConfig(
+        format="[%(asctime)s] [%(levelname)s] [%(filename)s:%(lineno)d:%(funcName)s] - %(message)s",
```

Comment:
also format can be an environment variable?

Reply:
need an open devops issue to discuss logging in depth separetely

Reply:
here: https://github.com/ai-cfia/membrane-backend/issues/58